modules/cnv-resuming-node-maintenance-web.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-maintenance-mode.adoc
+
+[id="cnv-resuming-node-maintenance-web_{context}"]
+= Resuming a node from maintenance mode in the web console
+
+Resume a node from maintenance mode using the :kebab: found on each node in the 
```

Comment:
Double-check that the kebab renders right.

Reply:
Leaving this here until the relevant kebab PR gets merged

Reply:
Fixed as per the contrib#icon guide update.

Reply:
Also fixed the same in the two live migration modules(+assemblies) that were merged in the partner PR #14922